src/cmd-cloud-prune

```diff
@@ -162,7 +174,24 @@ def main():
                         case "build":
                             prune_build(s3_client, bucket, prefix, build_id, args.dry_run)
                             pruned_build_ids.append(build_id)
-
+                        case "containers":
+                            # Only prune for x86_64 as it's required for all builds, covering all architectures.
+                            if arch == "x86_64":
+                                if build_id in barrier_releases:
+                                    print(f"Release {build_id} is a barrier release, keeping.")
```

Comment:
Let's include which action we are skipping in the log message. Let's
also add a comment with some context on why we are doing this:

```suggestion
                                    # Since containers are used for updates we need
                                    # to keep around containers for barrier releases.
                                    print(f"Release {build_id} is a barrier release. Skipping container prune.")
```

Reply:
fixed in new upload